migrations/20200927132044-update-patient-barcode-non-nullable.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.query(
+      `UPDATE patients SET "barcodeValue" = "id" WHERE "barcodeValue" IS NULL`
+    );
+    return queryInterface.changeColumn('patients', 'barcodeValue', {
+      type: Sequelize.STRING,
+      allowNull: false,
+      unique: true,
+    });
+  },
+  down: async (queryInterface, Sequelize) => {
+    await Promise.all([
+      queryInterface.changeColumn('patients', 'barcodeValue', {
+        type: Sequelize.STRING,
+        allowNull: true,
+        unique: false,
+      }),
+      queryInterface.removeConstraint('patients', 'patients_barcodeValue_key'),
```

Comment:
i noticed that sequelize is kinda silly and didn't clean up the constraints when repeatedly running and undoing the migration, so i pushed this commit to fix that (so we hopefully make it faster for this PR to get merged)
![image](https://user-images.githubusercontent.com/59516223/94382636-7f7b7f00-010b-11eb-8045-493cee51a1ad.png)